tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -138,11 +138,22 @@ def test_compute_distance2line(packet_params, expected_stat, expected_dist, pack
     assert obtained_stat == expected_stat
 
 
-def test_compute_distance2continuum():
-    distance_to_electron = 4.359272608766106e+28
-    tests.test_compute_distance2continuum.restype = c_double
-    np.testing.assert_almost_equal(tests.test_compute_distance2continuum(),
-                                  distance_to_electron)
+@pytest.mark.parametrize(
+    ['packet_params', 'model_params', 'expected_params'],
+    [({'virtual_packet': 0}, {'cont_status': 0}, {'chi_cont': 6.652486e-16, 'd_cont': 4.359272608766106e+28}),
+     ({'virtual_packet': 0}, {'cont_status': 1}, {'chi_cont': 6.652486e-16, 'd_cont': 4.358643523052283e+28}),
+     ({'virtual_packet': 1}, {'cont_status': 0}, {'chi_cont': 6.652486e-16, 'd_cont': 1e+99}),
+     ({'virtual_packet': 1}, {'cont_status': 1}, {'chi_cont': 6.652486e-16, 'd_cont': 1e+99})]
+)
+def test_compute_distance2continuum(packet_params, model_params, expected_params, packet, model):
```

Comment:
Thanks for writing a test for that method. Currently continuum interaction is a feature in development. I'd say it is safe for now to only test `cont_status=0`. @chvogl Is there any reason to test `cont_status=1` in the current master?